configs/nosite-x86_64/variables.yaml

```diff
@@ -4,9 +4,12 @@
 # SPDX-License-Identifier: Apache-2.0
 
 variables:
-  batch_time: ''
-  mpi_command: 'mpirun -n {n_nodes} -c {n_ranks} --oversubscribe'
-  batch_submit: '{execute_experiment}'
-  batch_nodes: ''
-  batch_ranks: ''
-  batch_timeout: ''
+  n_ranks: '1000001'  # placeholder value
+  n_nodes: '1000001'  # placeholder value
+  scheduler: "sierra"
```

Comment:
The scheduler on Sierra is LSF (the full name is IBM's Spectrum LSF).

Docs here if you need/are interested:
https://hpc.llnl.gov/documentation/tutorials/using-lc-s-sierra-systems#Monitor